src/App.vue

```diff
@@ -303,16 +303,23 @@ export default class App extends Mixins(
         // Only set Future Effective and Priority to Special Resolution Fee
         const specialResolutionFilingData = filingData.find(x => x.filingTypeCode === FilingCodes.SPECIAL_RESOLUTION)
         if (specialResolutionFilingData) {
-          specialResolutionFilingData.futureEffective = this.getEffectiveDateTime.isFutureEffective
-          specialResolutionFilingData.priority = this.getStaffPaymentStep.staffPayment.isPriority
+          if (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE) {
+            filingData.forEach(x => {
+              x.waiveFees = true
+            })
+          } else {
+            specialResolutionFilingData.futureEffective = this.getEffectiveDateTime.isFutureEffective
+            specialResolutionFilingData.priority = this.getStaffPaymentStep.staffPayment.isPriority
+          }
         }
-        filingData.forEach(x => {
-          x.waiveFees = (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE)
-        })
       } else if (this.getFilingData[0]) {
-        filingData[0].futureEffective = this.getEffectiveDateTime.isFutureEffective
-        filingData[0].priority = this.getStaffPaymentStep.staffPayment.isPriority
-        filingData[0].waiveFees = (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE)
+        // Avoid waiveFee with priority or futureEffective in the same request
+        if (this.getStaffPaymentStep.staffPayment.option === StaffPaymentOptions.NO_FEE) {
+          filingData[0].waiveFees = true
+        } else {
+          filingData[0].futureEffective = this.getEffectiveDateTime.isFutureEffective
+          filingData[0].priority = this.getStaffPaymentStep.staffPayment.isPriority
+        }
```

Comment:
I realized that the UI was sending `waiveFees` and `priority` flags at the same time to  the API, then I changed it.

Reply:
Good catch! I, too, find things like this thanks to the unit tests. 👍

Reply:
OK, I see the warnings still display for unknown custom element affix. I found that you can fix this warning by "using" Affix just like Vuetify... but this uncovers a new warning/error (which I didn't chase down).
Eg,
```
import Affix from 'vue-affix'
Vue.use(Affix)
...

Reply:
^^ the offsetHeight error is because affix can't find the relative element.

https://github.com/eddiemf/vue-affix/blob/12d7dc22ee8bc17945067cf6c2781847f8a1d254/src/affix.vue#L207

Reply:
^^ the affix component correctly gets the relativeElement prop in the tests, but this call returns null:
`document.querySelector(this.relativeElementSelector);`

👀

Reply:
I saw your comment about Friday... closing this conversation.